code/src/components/AlbumCover.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export const AlbumCover = (props) => {
+  return (
+    <div className="album-cover-overlay">
+
+      <a
+        href={props.link}
+        target="_blank"
+        rel="noreferrer"
```

Comment:
Nice touch that you added a link to album from the whole cover image as well as from the album name